product_docs/docs/pgd/5/rel_notes/pgd_5.1.0_rel_notes.mdx

```diff
@@ -0,0 +1,42 @@
+navTitle: "Version 5.1.0"
+---
+
+EDB Postgres Distributed version 5.1.0 is a minor version of EDB Postgres Distributed.
+This version addresses security vulnerabilities in dependencies for PGD-Proxy and PGD-CLI.
+
+!!! Note
+    This version is required for EDB Postgres Advanced Server versions 12.15, 13.11, 14.8 and later.
+
+| Component | Version | Type            | Description                                          |
+|-----------|---------|-----------------|------------------------------------------------------|
+| PGD       | 5.1.0   | Feature         | Add pid to the log message emitted upon writer process death (BDR-2534) |
+| PGD       | 5.1.0   | Feature         | Add group name in the bdr.event_summary view (BDR-3315) |
+| PGD       | 5.1.0   | Feature         | Add SLES support (BDR-3101)   SLES 15sp4 is now supported |
+| PGD       | 5.1.0   | Feature         | Add subscription status column to the group subscription summary view. (BDR-2198)  This will help distinguish whether NULL values are due to a node being down or a subscription being disabled.|
+| PGD       | 5.1.0   | Feature         | Add NOT group filter to Group Commit (BDR-3339)  Allows inverting the meaning of group filter to include all nodes except the ones in specifed group(s).|
+| PGD       | 5.1.0   | Feature         | Add bdr.drop_node_group (BDR-3313)  Allows dropping empty node groups.|
+| PGD       | 5.1.0   | Feature         | Add SYNCHRONOUS_COMMIT to Commit Scopes (BDR-3340)  This allows dynamic synchronous_commit-like behavior for replication.|
+| PGD       | 5.1.0   | Feature         | Add "event_node_name" column to bdr.event_summary (BDR-3317) |
+| PGD       | 5.1.0   | Feature         | Add write leader election to event history (BDR-3350) Add information about the node which is elected as a write leader for each group in the event_history catalog. While at it improve reporting of raft instance id in event_detail of event_history.|
+| PGD       | 5.1.0   | Feature         | Allow exporting and importing other instances' Raft snapshots.  Allow exporting and importing snapshots also for other instances and not only top raft instances.|
+| PGD       | 5.1.0   | Bug fix         | Fix memory leak in consensus process (BDR-3409, RT91830)  The memory consumed by the node is merely 32 bytes but when the consensus worker handles hundreds of requests per second, sustained for hours, the memory builds up. We saw 47% of memory consumed by consensus worker when used with HARP, which executes bdr.consensus_kv_fetch() at a rate of 600 times per second. |
+| PGD       | 5.1.0   | Bug fix         | Fix issue where a node can be inconsistent with the group after rejoining If a node had been part of a sub-group, then parted and then rejoined to the group, it could be inconsistent with the group. The changes from some nodes of the group would be replayed from a wrong starting point, resulting in potential data-loss. |
```

Comment:
```suggestion
| PGD       | 5.1.0   | Bug fix         | Fix issue where a node can be inconsistent with the group after rejoining. If a node had been part of a sub-group, then parted and then rejoined to the group, it could be inconsistent with the group. The changes from some nodes of the group would be replayed from a wrong starting point, resulting in potential data-loss. |
```